frontend/src/components/common/form/ApplicationSelect.vue

```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
```

Comment:
I feel it's okay to put this file under common, no need to create a sub form folder, thanks!!

Reply:
Ok, I'll refactor it. Thanks.